src/game/Tactical/Interface_Items.cc

```diff
@@ -2877,7 +2877,7 @@ void BeginKeyRingItemPointer( SOLDIERTYPE *pSoldier, UINT8 ubKeyRingPosition )
 	}
 	else
 	{
-		RemoveKeyFromSlot( pSoldier, ubKeyRingPosition, &gItemPointer );
+		RemoveKeysFromSlot( pSoldier, ubKeyRingPosition, 1, &gItemPointer );
 		fOk = (gItemPointer.ubNumberOfObjects == 1);
 	}
```

Comment:
Can't this else block be merged with the previous one now? From what I can tell there is only the difference in the count of items.

Reply:
Sure. I'll also remove the nonsensical comment about a bloodied knife while I am here.